api/SVGAnimatedString.json

```diff
@@ -76,10 +76,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "6.1"
```

Comment:
Looks like this is the only SVGAnimated* interface which has the `animVal` and `baseVal` entries, even though there are 12 interfaces that have them in the SVG spec's IDL.

Anyway, I don't trust this version, Safari 6.1 keeps coming up as the version for added attributes, and I think it's because we're testing the prototype only. The interface without these attributes wouldn't make any sense. Can you double check?

Reply:
Yeah, we're testing the prototype rather than an instance (I'm not yet certain how to create an instance for these).  Let me revert changes that set "6.1" for attributes.

Reply:
Thanks!